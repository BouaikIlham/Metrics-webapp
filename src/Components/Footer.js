import React from 'react';

const space = ' ';

const Footer = () => (
    <footer>
        <a href="https://github.com/BouaikIlham">
            Copyright Â©IlhamBouaik
            {space + new Date().getFullYear()}
        </a>
    </footer>
);

export default Footer;