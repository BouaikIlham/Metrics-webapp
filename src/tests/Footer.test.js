import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import Footer from '../Components/Footer';
import store from '../redux/configStore';

describe('Footer tests', () => {
    test('should render', () => {
        expect(Footer).toMatchSnapshot();
    });

    test('Checks if footer content is there by checking text', () => {
        render(
            <Provider store={store}>
                <BrowserRouter>
                    <Footer />
                </BrowserRouter>
                ,
            </Provider>
        );
    });
});