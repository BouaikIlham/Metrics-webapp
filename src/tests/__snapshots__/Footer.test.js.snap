// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests should render 1`] = `[Function]`;
